packages/platform/src/Http/Headers.ts

```diff
@@ -141,3 +142,14 @@ export const remove: {
   (key: string) => (self: Headers) => Headers,
   (self: Headers, key: string) => Headers
 >(2, (self, key) => ReadonlyRecord.remove(self, key.toLowerCase()) as Headers)
+
+/**
+ * @since 1.0.0
+ */
+export const redact: {
+  (key: string): (self: Headers) => Record<string, string | Secret.Secret>
```

Comment:
I think we should support a ReadonlyArray, as you can't chain this function currently